src/main/resources/org/sagebionetworks/web/client/view/SearchViewImpl.ui.xml

```diff
@@ -37,16 +37,18 @@
 						styleName="margin-top-10" />
 				</div>
 				<g:HTMLPanel ui:field="narrowResultsPanel"
-					addStyleNames="col-md-3 col-md-pull-9">
+					addStyleNames="col-md-3 col-md-pull-9" visible="false">
 					<div class="well">
 						<h5>Narrow Results</h5>
-						<bh:Italic>Searching for a user profile? Try our </bh:Italic>
-						<b:Anchor href="#!PeopleSearch:"
-							addStyleNames="margin-left-5 font-italic">People Search</b:Anchor>
-						<bh:Italic>.</bh:Italic>
 						<g:SimplePanel ui:field="facetPanel" />
 					</div>
 				</g:HTMLPanel>
+				<bh:Div addStyleNames="col-md-3 col-md-pull-9">
```

Comment:
Pull out the PeopleSearch link from the Narrow Results panel.  Will be shown below facets if they are defined.